src/components-nextjs/Sitemap/createPaths.ts

```diff
@@ -0,0 +1,21 @@
+import type { Category, Story } from '@prezly/sdk';
+
+export function createPaths(stories: Story[], categories: Category[]) {
+    const storiesUrls = stories.map(({ slug }) => `/${slug}`);
+    const categoriesUrls = categories
+        .map((category) => {
+            const translations = Object.values(category.i18n);
+            const allSlugs = translations
+                .map(({ slug }) => slug || '')
+                .filter(Boolean)
+                .reduce(
+                    (slugs, slug) => (slugs.includes(slug) ? [...slugs] : [...slugs, slug]),
+                    [] as string[],
```

Comment:
I think you can do `.reduce<string[]>(` instead of the typecast.

Reply:
Agree, will check!